iXBRLViewerPlugin/viewer/src/js/fact.js

```diff
@@ -262,31 +266,49 @@ export class Fact {
         return this.f.err == 'INVALID_IX_VALUE';
     }
 
+    getAccuracyLabel(value, isScale=false) {
+        return this._report.getAccuracyLabel(
+                isScale ? -value : value, // Invert value if this is a scale value
+                this.isMonetaryValue(),
+                this.report().qname(this.measure()).localname
+        );
+    }
+
     readableAccuracy() {
         if (!this.isNumeric() || this.isNil()) {
             return i18next.t("common.notApplicable");
         }
-        let d = this.decimals();
+        const d = this.decimals();
         if (d === undefined) {
             return i18next.t("common.accuracyInfinite")
         }
         else if (d === null) {
             return i18next.t("common.unspecified");
         }
-        var name = i18next.t(`currencies:accuracy${d}`, {defaultValue:"noName"});
-        if (this.isMonetaryValue()) {
-            var currency = this.report().qname(this.measure()).localname;
-            if (d == 2) {
-                var name = i18next.t(`currencies:cents${currency}`, {defaultValue: name});
-            }
+        const label = this.getAccuracyLabel(d);
+        if (label != null) {
+            return `${d} (${label})`;
+        }
+        else {
+            return d.toString();
+        }
+    }
+
+    readableScale() {
+        if (!this.isNumeric() || this.isNil()) {
+            return i18next.t("common.notApplicable");
+        }
+        const scale = this.scale();
+        if (scale === undefined || scale === null) {
+            return i18next.t("common.unspecified");
         }
-        if (name !== "noName") {
-            d += " ("+name+")";
+        const label = this.getAccuracyLabel(scale, true);
```

Comment:
Only in the specific case of monetary values with `decimals="-2"` and `scale="2"`, and we're effectively saying "accurate to the nearest cent, reported in cents" which seems reasonable.

Reply:
Ah, I finally understand now, thanks :-).

There is something confusing about using "accuracy" as the i18n key for describing "scale".  It seems like it might be clearer to invert the keys (accuracy described in terms of scale labels) or to use some other word (exponent?).

Reply:
I [agree](https://github.com/Workiva/ixbrl-viewer/pull/425#discussion_r1198191696) "scale" is the more intuitive label to use over "accuracy". That's what I'll go with unless anyone feels strongly that something more general like `exponent3` or `decimalPlace10^3` is better.

Reply:
"scale" WFM.